scripts/plugins/deploy.py

```diff
@@ -0,0 +1,65 @@
+import yaml
+import os
+import subprocess
+import envbash
+
+# Load plugin.config.yml
+bitops_dir=os.environ['BITOPS_DIR']
+with open(bitops_dir+'/plugin.config.yml', 'r') as stream:
+    try:
+        plugins_yml = yaml.load(stream, Loader=yaml.FullLoader)
+    except yaml.YAMLError as exc:
+        print(exc)
+
+# Loop through plugins and invoke each
+plugins_dir = bitops_dir + '/scripts/plugins/'
+operations_dir = os.environ['ENVROOT']
+for plugin in plugins_yml.get('plugins'):
+    plugin_name = plugin['name']
+
+    # Set ENV vars
+    plugin_dir = plugins_dir + plugin_name
+    os.environ['PLUGIN_DIR'] = plugin_dir
+    environment_dir = operations_dir + '/' + plugin_name
+    os.environ['ENVIRONMENT_DIR'] = environment_dir
```

Comment:
As discussed, because Hem is full of custom use cases (calling multiple helm charts in a single plugin), we're going to bake all this logic in to the Helm plugin itself. 

Long term, we'll create a more complex lifecycle than just
* before deploy
* deploy
* after deploy

To address use cases like this
https://github.com/bitovi/bitops/issues/58